src/formElementPlugins/checkbox/src/ApplicationView.tsx

```diff
@@ -164,13 +164,13 @@ type KeyMap = {
 }
 
 export const getCheckboxStructure = (
-  initialValue: CheckboxSavedState,
+  initialValue: CheckboxSavedState | null,
   checkboxes: Checkbox[],
   keyMap: KeyMap | undefined,
   isFirstRender: boolean
 ) => {
   // Returns a consistent array of Checkbox objects, regardless of input structure
-  const { values: initValues } = initialValue
+  const { values: initValues } = initialValue ?? {}
```

Comment:
The `currentReponse` actually had a more accurate typing (`initalValue` was `any`), so I had to add a few catches for the `null` case in a couple of places -- which is better overall.